Index.html

```diff
@@ -4,18 +4,26 @@
 <head>
     <title>Evaluations</title>
 
+    <!-- CSS -->
+
+    <link rel="stylesheet/css" type="text/css" href="assets/css/normalize.css" />
```

Comment:
please use the less @import directive to combine all those files. Only one less file should be added to the html page